eahub/templates/eahub/profile.html

```diff
@@ -124,113 +145,128 @@ <h1 class="col col-auto">{{ profile.name }}</h1>
         <div class="prof__summary">
           {{ profile.summary|default:''|urlize|linebreaks }}
 
-          <div class="row summary__sections">
-            {% if profile.offering %}
```

Comment:
offering and looking for not on same height

![Uploading image.png…]()

Reply:
fixed